src/main.rs

```diff
@@ -1,16 +1,35 @@
 use clap::Parser;
+use std::fs::File;
 use std::io::{self, BufRead, Seek, Write};
 use std::{fs, path};
 
+use std::io::BufReader;
+
+extern crate serde_with;
+extern crate serde_json;
+extern crate serde;
+use serde::{Deserialize, Serialize};
+#[derive(Debug, Deserialize, Serialize)]
+#[serde_with::skip_serializing_none]
+struct Config{
+    input: Option<String>,
+    output: Option<String>,
+    lang: Option<String>
+}
+
 const HTML_TEMPLATE: &str = "<!DOCTYPE html>\n<html lang=\"{{lang}}\">\n<head>\n\t<meta charset=\"UTF-8\">\n\t<meta http-equiv=\"X-UA-Compatible\" content=\"IE=edge\">\n\t<meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\n\t<title>\n\t\t{{title}}\n\t</title>\n</head>\n<body>\n";
 const DEFAULT_OUTPUT_DIR: &str = "./dist";
 
-#[derive(Parser)]
+#[derive(Parser, PartialEq, Default)]
```

Comment:
```suggestion
#[derive(Parser)]
```
These are not being used